### ISSUE TITLE
Window Border expands when showTabsInTitlebar is enabled

### ISSUE THREAD

**Issue by RRitts** [microsoft/terminal#2333]
# Environment

```none
Windows build number: 10.0.18362.239
Windows Terminal version: 0.3.2171.0
```

# Steps to reproduce

Set showTabsInTitlebar to true.

# Expected behavior

![working](https://devblogs.microsoft.com/commandline/wp-content/uploads/sites/33/2019/07/title-bar-v03.png)

# Actual behavior

![true](https://i.imgur.com/o9DtNfD.png)

When showTabsInTitlebar is false

![false](https://i.imgur.com/4eU4jLY.png)

**Comment by DHowett-MSFT** [comment]
Wow! This is hilarious/terrible.
What does your display topology look like? Multiple monitors? different scaling factors?

**Comment by RRitts** [comment]
It was a pretty unpleasant surprise

I'm running 3 Displays, one laptop and 2 external. All are at 1080 with 100% scaling.

**Comment by zadjii-msft** [comment]
This is actually #2057 which should be fixed by #2150. IIRC we decided that it didn't make the cut for 0.3, but it should be in the next release.
